packages/video_player/video_player/lib/video_player.dart

```diff
@@ -445,6 +474,33 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _applyVolume();
   }
 
+  /// Sets the playback speed of [this].
+  ///
+  /// [speed] indicates a speed value with different platforms accepting
+  /// different ranges for speed values (that must not be negative).
+  /// The values will be handled as follows:
+  /// * On web, no exceptions will be thrown, however, different browser
+  ///   implementations will e.g. mute audio.
+  ///   "Gecko mutes the sound outside the range `0.25` to `5.0`" (see https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/playbackRate).
+  /// * On iOS, you can sometimes not go above `2.0` playback speed on a video.
+  ///   An error will be thrown for if the option is unsupported. It is also
+  ///   possible that your specific video cannot be slowed down, in which case
+  ///   the plugin also reports errors.
+  /// * On Android, the handling is similar to web, where the plugin will not
+  ///   report any errors, however, you cannot expect support for all speed
+  ///   values (even though it should go way above `2.0`).
+  Future<void> setPlaybackSpeed(double speed) async {
+    if (speed < 0) {
```

Comment:
`if (speed <= 0) {`...

Reply:
I decided to add an extra case for that, just to explain it a little bit.